enterprise/server/test/integration/remote_execution/rbetest/rbetest.go

```diff
@@ -90,6 +91,10 @@ const (
 	defaultWaitTimeout = 20 * time.Second
 )
 
+var (
+	enableDocker = flag.Bool("rbetest.enable_docker", false, "Whether to enable docker for executors created via the rbetest framework")
```

Comment:
since we're on the verge of switching to podman, can we find a way to use that instead?

Reply:
Removed this flag for now since it's just for convenience when manual testing. Will handle podman as a separate PR, to keep this one small